.env.example

```diff
@@ -85,8 +85,9 @@ GCP_STATIC_BUCKET_NAME=
 GCP_PROJECT_ID=
 GCP_BUCKET_LOCATION=
 
-SITE_NAME=
-EMAIL_SIGNATURE=
-FOOTER_MANAGED_BY=
-FOOTER_SUPPORTED_BY=
+SITE_NAME="DataShare"
```

Comment:
@tompollard , perhaps a comment should be added above the new variables as I don't think `# Variables required to be set when storage type is set to GCP` applies to these new variables.

Reply:
Good idea, thanks for the suggestion. I added `# Site-specific content` as the header (best I could come up with for now, but hopefully it does the job for now and we can tweak later!)